src/sailor/admin.lua

```diff
@@ -0,0 +1,333 @@
+<td><input type="text"  name="default_error404" value="<%= conf.sailor.default_error404 %>" /></td>
+</tr>
+
+<tr>
+<td>Database Environment</td>
+<td><input type="text"  name="environment" value="<%= conf.sailor.environment %>"/></td>
+</tr>
+
+<tr>
+<td>Default Controller</td>
+<td><input type="text"  name="default_controller" value="<%= conf.sailor.default_controller %>"/></td>
+</tr>
+
+<tr>
+<td>Default Action</td>
+<td><input type="text"  name="default_action" value="<%= conf.sailor.default_action %>"/></td>
+</tr>
+
+<tr>
+<td>Default Static</td>
+<td><input type="text"  name="default_static" value="<%= tostring(conf.sailor.default_static) %>" /></td>
+</tr>
+
+<tr>
+<td>Toggle Autogen</td>
+<td><input type="radio" name="enable_autogen" value="true">True
+<input type="radio" name="enable_autogen" value="false" checked>False</td>
+</tr>
+
+<tr>
+<td>Toggle Admin</td>
+<td><input type="radio" name="enable_admin" value="true">True
+<input type="radio" name="enable_admin" value="false" checked>False</td>
+</tr>
+
+<tr>
+<td>Hide Stack Trace</td>
+<td><input type="radio" name="hide_stack_trace" value="true" checked>True
+<input type="radio" name="hide_stack_trace" value="false">False</td>
+</tr>
+
+<tr>
+<td>Toggle Friendly Urls</td>
+<td><input type="radio" name="friendly_urls" value="true">True
+<input type="radio" name="friendly_urls" value="false" checked>False</td>
+</tr>
+
+</table>
+
+<h3> Database Settings</h3>
+
+<table style="width:100%">
+
+<tr>
+<td> Database Driver</td>
+<td><input type="text"  name="driver" value="<%= tostring(conf.db.development.driver) %>" /></td>
+</tr>
+
+
+<tr>
+<td> Host</td>
+<td><input type="text"  name="host" value="<%= tostring(conf.db.development.host) %>" /></td>
+</tr>
+
+
+<tr>
+<td> Username</td>
+<td><input type="text"  name="user" value="<%= conf.db.development.user %>" /></td>
+</tr>
+
+<tr>
+<td> Password</td>
+<td><input type="text"  name="pass" value="<%= conf.db.development.pass %>" /></td>
+</tr>
+
+<tr>
+<td> Database Name</td>
+<td><input type="text"  name="dbname" value="<%= conf.db.development.dbname %>" /></td>
+</tr>
+</table>
+
+<h3> Mail Server Settings</h3>
+
+<table style="width:100%">
+
+<tr>
+<td>Mail Server</td>
+<td><input type="text"  name="smtpserver" value="<%= conf.smtp.server %>" /></td>
+</tr>
+
+<tr>
+<td> Username</td>
+<td><input type="text"  name="smtpuser" value="<%= conf.smtp.user %>" /></td>
+</tr>
+
+<tr>
+<td> Password</td>
+<td><input type="text"  name="smtppass" value="<%= conf.smtp.pass %>" /></td>
+
+<tr>
+<td> From Address</td>
+<td><input type="text"  name="smtpfrom" value="<%= conf.smtp.from %>" /></td>
+</tr>
+
+<tr>
+<td> <u>Lua at Client VM</u></td>
+<td><input type="text"  name="lua_at_client" value="<%= conf.lua_at_client.vm %>" /></td>
+</tr>
+
+<tr>
+<td><u>Toggle Debug Mode</u></td>
+<td><input type="radio" name="debug" value="true">True
+<input type="radio" name="debug" value="false" checked>False</td>
+</tr>
+
+</table>
```

Comment:
Wouldn't it be better to put details in a table then loop through it to make the tables and the form fields? It would also be more maintainable